### PR TITLE
Allow igenomes to work

### PR DESCRIPTION
**Pull request by SPPearce**
Fixing the template to allow igenomes to work.
Currently the parameters e.g. `fasta` are being set to a default of `null` in the nextflow.config, which means that they are not able to be overwritten via the script that tries to pull them from the igenomes config.

We might want to add a test profile that uses igenomes; we could swap the test_full to do this.

### CODE REVIEW

**Comment by SPPearce** [comment]
@nh13 , is there anything else we need to do before a v1.0.1 release?

File: nextflow.config [hunk 1]
```diff
@@ -73,8 +69,8 @@ params {
 
     // Schema validation default options
     validationFailUnrecognisedParams = false
-    validationLenientMode            = false
-    validationSchemaIgnoreParams     = 'genomes,igenomes_base,monochromeLogs'
+    validationLenientMode            = true
```
Comment (nh13) [inline nextflow.config:72]: Why true?

**Review by nh13 — CHANGES_REQUESTED** [review]
I am confused why we make validation lenient.

File: nextflow.config (hunk 1 above)
Comment (SPPearce) [reply nextflow.config:72]: That is a leftover from fixing the other bug with the string/integer

Comment (nh13) [reply nextflow.config:72]: Perhaps we revert it back since it's no longer needed.
```suggestion
    validationLenientMode            = false
```

**Review by nh13 — APPROVED** [review]
LGTM, thank-you!

**Pull request merged**